specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json

```diff
@@ -317,7 +317,7 @@
               "Deleted"
             ],
             "x-ms-enum": {
-              "name": "databaseState",
+              "name": "longTermRetentionDatabaseState",

```

Comment:
While we're at it consider creating a common definition for this parameter in the `parameters`/`definitions` section thus avoiding duplication

Reply:
We have swagger generator so for now it's easier to keep it duplicated rather than manually edit :)